### DISCUSSION TITLE
Manufactoring order quantity is not integer

### DISCUSSION THREAD

**Discussion by BigDanTheOne** [frePPLe/frepple#661] [Q&A] [answered]
Hello! I need help with frepple planning. When I plan my sales orders in frepple I get fractional quantity of operation. This also leads  to fractional quantity of items.
<img width="1175" alt="image" src="https://github.com/frePPLe/frepple/assets/45150310/05fea954-72a5-4318-89e4-6fefdade6629">

All my operations are time_fixed
<img width="1033" alt="image" src="https://github.com/frePPLe/frepple/assets/45150310/9ff900aa-6f05-41fb-bcf3-040521121668">
I want frepple to plan only integer quantity of manufactoring orders and items. How can I obtain this?

**Comment by jdetaeye** [comment, marked as answer]
Have a look at the "operation.size_multiple" field.   Just set it to 1.

**Reply by BigDanTheOne** [reply]
> Have a look at the "operation.size_multiple" field. Just set it to 1.

It worked, thank you very much! Now my manufactoring order quantity are integers.
May I ask you few more questions if you don't mind:
I don't understand how does downstream / upstream operations work. Take a look at the scressnshots:
<img width="832" alt="image" src="https://github.com/frePPLe/frepple/assets/45150310/84a08833-a75e-41e4-85e0-a93630b36304">
<img width="930" alt="image" src="https://github.com/frePPLe/frepple/assets/45150310/315700da-e536-4a08-afa7-bdc8f686bf31">

1) What does quantity mean?
2) why does it display that from MO "5" to MO "18" I pass 2/2 and 2/2 of smth and in MO "18" I receive 1/1 and 2/3 of smth?

**Comment by jdetaeye** [comment]
Please use a separate discussion for different unrelated topics.   Only in this way are the answers usable and interesting to others as well.  

The quantities reflect the pegging of materials flow between operations. 
The "downstream operations" allows you to follow how a certain produced quantity is being consumed by other manufacturing and ultimately ends up in a sales order for an end item.
The "upstream operations" allows you to trace where the consumes material came from that a sales order or manufacturing order.
